app/src/main/res/layout/activity_local_file_transfer.xml

```diff
@@ -4,174 +4,173 @@
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
-  android:fitsSystemWindows="true"
   android:background="@android:color/transparent"
-  tools:context=".zim_manager.local_file_transfer.LocalFileTransferActivity"
-  >
+  android:fitsSystemWindows="true"
+  tools:context=".zim_manager.local_file_transfer.LocalFileTransferActivity">
 
   <com.google.android.material.appbar.AppBarLayout
     android:id="@+id/app_bar_local_file_transfer"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    app:layout_constraintTop_toTopOf="parent"
     app:layout_constraintLeft_toLeftOf="parent"
     app:layout_constraintRight_toRightOf="parent"
-    >
+    app:layout_constraintTop_toTopOf="parent">
 
     <androidx.appcompat.widget.Toolbar
       android:id="@+id/toolbar_local_file_transfer"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      app:titleTextColor="#FFFFFF"
       app:title="@string/local_zim_sharing"
-      ></androidx.appcompat.widget.Toolbar>
+      app:titleTextColor="#FFFFFF" />
 
   </com.google.android.material.appbar.AppBarLayout>
 
+  <View
```

Comment:
I think it would be nicer to have the 3 views that make up the chain come first, they are sort of a meta layout piece and then you should be able to read the view hierarchy as it is displayed. Them being interspersed is a bit harder to read I feel.

Reply:
[Done](https://github.com/kiwix/kiwix-android/pull/1538/commits/7ff72056b8ecadd3cec3aa9ce353e9b164ca9ebd)